README.md

```diff
@@ -13,7 +13,7 @@ Storybook: https://neetoui-rn.onrender.com/
 ## Installation
 
 ```
-yarn add @bigbinary/neetoui-rn react-native-svg react-native-webview @shopify/flash-list react-native-reanimated
+yarn add @bigbinary/neetoui-rn react-native-svg react-native-webview @shopify/flash-list react-native-reanimated react-native-fast-image
```

Comment:
@kamolesh97 also needs to add in 2 more places in getting started markdown file.
https://github.com/bigbinary/neeto-ui-rn/blob/master/docs/pages/1.getting-started.md